ftw/simplelayout/browser/resources/videoblock.js

```diff
@@ -12,7 +12,7 @@
   function resizeAll() { $.map($(".sl-youtube-video"), resizeYoutubePlayer); }
 
   function onPlayerReady(player) {
-    var iframe = player.target.a;
+    var iframe = player.target.f;
```

Comment:
This alone won't work. The bundle won't reload the file. We either have to create a second file where video block is compiled into or we have to load it into the default simplelayout bundle.

Reply:
@Nachtalb I don't really get it. But I'm don't know anyway how to deal with that the right i generated a new simplelayout-contenttypes-compiled.js and it worked locally in dev and prod mode. But I think you guys know what you're doing.

Reply:
What do you mean you generated a new simplelayou-contenttypes-compiled.js? This file does not exist...